electrumx/lib/atx.py

```diff
@@ -0,0 +1,32 @@
+from .tx import DeserializerSegWit, Deserializer
+
+
+class BTCV3KeysDeserializer(DeserializerSegWit):
+    def _read_tx_and_hash_segwit(self):
+        return DeserializerSegWit.read_tx_and_hash(self)
+
+    def _read_tx_and_hash_no_segwit(self):
+        return Deserializer.read_tx_and_hash(self)
+
+    def _check_if_alert_exist(self):
+        if self.binary_length > self.cursor:
+            return True
+        return False
+
+    def _check_if_alert_is_seg_wit(self):
```

Comment:
It's a convention to spell `segwit` together, not as `seg-wit`.

Reply:
done